src/packages/admin-ui-components/src/dropdown/dropdown.tsx

```diff
@@ -15,18 +16,21 @@ export interface DropdownItem {
 		| void
 		| boolean /** Event emitted when the dropdown item is clicked- return false to prevent closing the dropdown  */;
 	renderAfter?(): ReactNode /** Render function after the item */;
+	className?: string;
 }
 
 export interface DropdownProps extends Partial<ButtonProps> {
 	items: Array<DropdownItem> /** List of items in the dropdown */;
 	className?: string /** Make button look like textfield not button */;
 	defaultValue?: DropdownItem;
+	isDropup?: boolean /** Make dropdown appear above the button */;
```

Comment:
This could be `position='up'` ?

Reply:
yea but that's less fun... will implement.